File: user-service/src/main/java/com/project/user_service/config/SwaggerConfig.java
package com.project.user_service.config;

import org.springdoc.core.models.GroupedOpenApi;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class SwaggerConfig {

    @Bean
    public GroupedOpenApi customOpenApi(){
        return GroupedOpenApi.builder()
                .group("user-service")
                .packagesToScan("com.project.user_service.controller")
                .build();
    }

}

File: task-management-service/src/main/kotlin/com/project/task_management_service/config/SwaggerConfig.kt
package com.project.task_management_service.config

import org.springdoc.core.models.GroupedOpenApi
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class SwaggerConfig {
    @Bean
    fun customOpenApi(): GroupedOpenApi {
        return GroupedOpenApi.builder()
            .group("task-management-service")
            .packagesToScan("com.project.task-management-service.controller")
            .build()
    }
}